tests/props/test_from_props.py

```diff
@@ -83,3 +93,5 @@ def test_from_props(props, error_msg):
             assert error_msg in str(exc)
         else:
             assert False, exc
+    else:
+        assert not error_msg, f"Expected InvalidPropertiesError with msg='{error_msg}'."
```

Comment:
Good catch!

I'd assert `error_msg is None` here, that would leave us possibility of expecting `InvalidPropertiesError` with *any* mesage, when `error_msg == ""`.
```suggestion
        assert (
            error_msg is None
        ), f"Expected InvalidPropertiesError with msg including '{error_msg}'."
```
(weird formatting courtesy of `black`)